server/command.go

```diff
@@ -240,3 +189,86 @@ func (p *Plugin) ExecuteCommand(_ *plugin.Context, args *model.CommandArgs) (*mo
 		return p.getCommandResponse(args, "Unknown action, please use `/gitlab help` to see all actions available."), nil
 	}
 }
+
+// SubscribeCommand proccess the /gitlab subscribe command.
+// It returns a message and handles all errors my including helpfull information in the message
+func (p *Plugin) subscribeCommand(parameters []string, channelID string, config *configuration, info *gitlab.GitlabUserInfo) string {
+
+	features := "merges,issues,tag"
+
+	txt := ""
+	if len(parameters) == 0 {
+		return "Please specify a repository or 'list' command."
+	} else if len(parameters) == 1 && parameters[0] == "list" {
+		subs, err := p.GetSubscriptionsByChannel(channelID)
+		if err != nil {
+			txt = err.Error()
+			return txt
+		}
+
+		if len(subs) == 0 {
+			txt = "Currently there are no subscriptions in this channel"
+		} else {
+			txt = "### Subscriptions in this channel\n"
+		}
+		for _, sub := range subs {
+			txt += fmt.Sprintf("* `%s` - %s\n", strings.Trim(sub.Repository, "/"), sub.Features)
+		}
+		return txt
+	} else if len(parameters) > 1 {
+		features = strings.Join(parameters[1:], " ")
+	}
+
+	// Resolve namespace and project name
+	fullPath := normalizePath(parameters[0], config.GitlabURL)
+
+	namespace, project, err := p.GitlabClient.
+		ResolveNamespaceAndProject(info, fullPath, config.EnablePrivateRepo)
+	if err != nil {
+		if errors.Is(err, gitlab.ErrNotFound) {
+			return "Resource with such path is not found."
+		} else if errors.Is(err, gitlab.ErrPrivateResource) {
+			return "Requested resource is private."
+		}
+		p.API.LogError(
+			"unable to resolve subscription namespace and project name",
+			"err", err.Error(),
+		)
+		return err.Error()
+	}
+
+	// Create subscription
+	if subscribeErr := p.Subscribe(info, namespace, project, channelID, features); err != nil {
```

Comment:
I think `subscribeErr` is meant to be used here
```suggestion
	if subscribeErr := p.Subscribe(info, namespace, project, channelID, features); subscribeErr != nil {
```